src/main/java/com/puppycrawl/tools/checkstyle/checks/OrderedPropertiesCheck.java

```diff
@@ -229,7 +229,7 @@ private static class SequencedProperties extends Properties {
          * Returns a copy of the keys.
          */
         @Override
-        public synchronized Enumeration<Object> keys() {
```

Comment:
It looks like an ordinary copy'n'paste bug. There is nothing to synchronize in this method.